model-server/src/test/kotlin/org/modelix/model/server/ModelServerTestUtil.kt

```diff
@@ -38,7 +41,13 @@ suspend fun ApplicationTestBuilder.createModelClient(): ModelClientV2 {
 
 fun Application.installDefaultServerPlugins() {
     install(WebSockets)
-    install(ContentNegotiation) { json() }
+    install(ContentNegotiation) {
+        json()
+        register(
```

Comment:
Yes, I can try to figure out if I can generate a method for that automatically. That's a bit annoying because every data type that we use needs one of these lines now.

Reply:
Does the pushed approach look right to you?

Reply:
Looks good in general.

It migth be an issue, that `registerJsonTypes` is an extension function.
How can I call extension functions from different packages (e. g. `org.modelix.api.v2.Paths.registerJsonType` and `org.modelix.api.v1.Paths.registerJsonType`)?

Reply:
You need import aliases or calls without imports then. I couldn't find an easy version to include the file name in the function name and even without being an extension function, we would then have multiple imports to the same name if we need to register also for v1 later on (or v3).